spring-kafka-2/src/main/java/com/baeldung/spring/kafka/multiplelisteners/BookConsumer.java

```diff
@@ -0,0 +1,26 @@
+package com.baeldung.spring.kafka.multiplelisteners;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookConsumer {
+    private static final Logger LOGGER = LoggerFactory.getLogger(BookConsumer.class);
+
+    @KafkaListener(topics = "books", groupId = "books-content-search")
+    public void bookContentSearchConsumer(BookEvent event) {
+        LOGGER.info(String.format("Books event received for full-text search indexing => %s", event.toString()));
```

Comment:
Instead of using String.format is better for performance issues to use placeholder for variable so in this case:

`logger.info("Books event received for full-text search indexing => {}", event);`

Please do the same for the rest of log entries.